packages/advanced-logic/test/utils/payment-network/meta-pn-data-generator.ts

```diff
@@ -0,0 +1,448 @@
+import * as TestData from '../test-data-generator';
+
+import { ExtensionTypes, IdentityTypes, RequestLogicTypes } from '@requestnetwork/types';
+import * as AnyToErc20Create from './erc20/any-to-erc20-proxy-create-data-generator';
+import * as AnyToErc20Add from './erc20/any-to-erc20-proxy-add-data-generator';
+
+export const arbitraryTimestamp = 1544426030;
+
+// ---------------------------------------------------------------------
+// Mock addresses for testing ETH payment networks
+export const paymentAddress = '0x627306090abaB3A6e1400e9345bC60c78a8BEf57';
+export const refundAddress = '0xf17f52151EbEF6C7334FAD080c5704D77216b732';
+export const feeAddress = '0xC5fdf4076b8F3A5357c5E395ab970B5B54098Fef';
+export const feeAmount = '2000000000000000000';
+export const invalidAddress = '0x not and address';
+export const tokenAddress = '0x6b175474e89094c44da98b954eedeac495271d0f';
+export const network = 'mainnet';
+export const saltMain = 'ea3bc7caf64110ca';
+export const salt1 = 'ea3bc7caf64110cb';
+export const salt2 = 'ea3bc7caf64110cc';
+export const salt3 = 'ea3bc7caf64110cd';
+export const salt4 = 'ea3bc7caf64110ce';
+// ---------------------------------------------------------------------
+export const baseParams = (salt: string) => ({
+  feeAddress,
+  feeAmount,
+  paymentAddress,
+  refundAddress,
+  salt,
+  acceptedTokens: [tokenAddress],
+  network,
+});
+export const extendedParams = (salt: string) => ({
+  ...baseParams(salt),
+  maxRateTimespan: undefined,
+  payeeDelegate: undefined,
+  payerDelegate: undefined,
+  paymentInfo: undefined,
+  receivedPaymentAmount: '0',
+  receivedRefundAmount: '0',
+  refundInfo: undefined,
+  sentPaymentAmount: '0',
+  sentRefundAmount: '0',
+});
+// actions
+export const actionCreationFull = {
```

Comment:
Can you try to rename the fixtures in clear way so we can understand why adding a bitcoin payment is expected to throw, for instance? Example in this case `createAnyToErc20Mainnet`.